samples/net/openthread/coap_server/README.rst

```diff
@@ -1,13 +1,14 @@
 .. _coap_server_sample:
 
-openthread: coap_server
-##################
+OpenThread: CoAP Server
+#######################
 
-The openthread coap_server sample demonstrates interactions between nodes performing different Thread roles with the use of OpenThread and CoAP protocol.
+The OpenThread CoAP Server sample demonstrates interactions between nodes performing different Thread roles with the use of OpenThread and CoAP protocol.
 
 Overview
 ********
-
+You can use this sample application as a starting point to implement a CoAP application.
+This sample uses native OpenThread CoAP API for communication. For example usage of Zephyr CoAp API refer to :ref:`coap_client_sample`
 
 Requirements
```

Comment:
All the comments from the client sample about this section are valid for server sample as well.